### DISCUSSION TITLE
Multiple duplicate screenshots

### DISCUSSION THREAD

**Discussion by mccmrunal** [ui5-community/wdi5#604] [Q&A]
We are using allure report to genereate the report but we observed that there are multiple duplicate screenshots, for which I think the cause is multiple click events getting fired on the call of press() function in wdi5 test case .
So  for visualizaiton purpose I have logged the commands from wdio afterCommand Hook
![image](https://github.com/ui5-community/wdi5/assets/47515609/06e7bf6b-a3ab-49ab-a5a9-b04dc48d4294)
![image](https://github.com/ui5-community/wdi5/assets/47515609/1ffa7fc7-0b49-4513-8eb7-cb7833a47ef6)
![image](https://github.com/ui5-community/wdi5/assets/47515609/07e6f0c9-8c70-425b-aa4d-e263060fbc5a)

This is the code part where I have logged the comments

So is there any possible solution so that we dont incur these multiple screenshot issue resulting in reduced report size

**Comment by vobu** [comment]
your code looks fine from a pure source code perspective.
yet i can only attest to that that we haven't seen the behaviour you describe previously... so my hunch is that there is a side effect happening somewhere.
